README.md

```diff
@@ -1,4 +1,5 @@
-[![Anaconda-Server Badge](https://anaconda.org/bsteubing/activity-browser/badges/version.svg)](https://anaconda.org/bsteubing/activity-browser)
+[![conda-forge version](https://img.shields.io/conda/vn/conda-forge/activity-browser.svg)](https://anaconda.org/conda-forge/activity-browser)
+[![conda-forge version](https://img.shields.io/conda/vn/bsteubing/activity-browser.svg)](https://anaconda.org/bsteubing/activity-browser)
 [![Anaconda-Server Badge](https://anaconda.org/bsteubing/activity-browser/badges/downloads.svg)](https://anaconda.org/bsteubing/activity-browser)
```

Comment:
Should be replaced with `[![Conda Downloads](https://img.shields.io/conda/dn/conda-forge/activity-browser.svg)](https://anaconda.org/conda-forge/activity-browser)`?

Reply:
Missed that, thanks.